recipes/protobuf/all/conanfile.py

```diff
@@ -229,9 +229,10 @@ def package_info(self):
                 self.cpp_info.components["libprotobuf"].defines = ["PROTOBUF_USE_DLLS"]
 
         # libprotoc
-        self.cpp_info.components["libprotoc"].set_property("cmake_target_name", "protobuf::libprotoc")
-        self.cpp_info.components["libprotoc"].libs = [lib_prefix + "protoc" + lib_suffix]
-        self.cpp_info.components["libprotoc"].requires = ["libprotobuf"]
+        if Version(self.version) >= "3.14.0" and self.settings.os != "tvOS":
```

Comment:
```suggestion
        if self.settings.os != "tvOS":
```

Reply:
I used the same condition we had in `generate()`. I guess it's because prior to 3.14.0 there is not libprotoc, but not sure. I tested that and it's fine to remove for newer versions.

https://github.com/conan-io/conan-center-index/blob/master/recipes/protobuf/all/conanfile.py#L100